DDTest/CMakeLists.txt

```
@@ -48,16 +48,20 @@ foreach(TEST_NAME
   set_tests_properties(t_${TEST_NAME} PROPERTIES FAIL_REGULAR_EXPRESSION "TEST_FAILED")
 endforeach()
 
-ADD_TEST( t_test_python_import "${CMAKE_INSTALL_PREFIX}/bin/run_test.sh"
-  pytest ${PROJECT_SOURCE_DIR}/DDTest/python/test_import.py)
-SET_TESTS_PROPERTIES( t_test_python_import PROPERTIES FAIL_REGULAR_EXPRESSION  "Exception;EXCEPTION;ERROR;Error" )
+find_program(HAVE_PYTEST pytest)
```

```suggestion
find_program(HAVE_PYTEST pytest)
if(NOT HAVE_PYTEST)
  message(WARNING "pytest not found! Skipping pytest tests.")
endif()
```
Avoid silently skipping tests